doc-content/enterprise-only/optimizer/quarkus-creating-proj-proc.adoc

```diff
@@ -0,0 +1,96 @@
+[id='proc-quarkus-creating-proj_{context}']
+
+
+= Creating the school timetable project
+
+The school timetable project lets you get up and running with {PLANNER} and Quarkus application using Apache Maven and the Quarkus Maven plug-in.
+
+.Procedure
+
+. In a command terminal, enter the following command to verify that Maven is using JDK {JDKVersion} and that the Maven version is {MavenVersion} or higher:
++
+[source]
+----
+mvn --version
+----
+
+. If the preceding command does not return JDK {JDKVersion}, add the path to JDK {JDKVersion} to the PATH environment variable and enter the preceding command again.
+
+. To generate the project, enter one of the following commands:
++
+NOTE: Apple macOS and Microsoft Windows are not supported production environments.
++
+* If you are using Linux or Apple macOS, enter the following command:
++
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:1.13.1.Final:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=optaplanner-quickstart \
+    -Dextensions="resteasy,resteasy-jackson,optaplanner-quarkus,optaplanner-quarkus-jackson" \
+    -DnoExamples
+cd optaplanner-quickstart
+----
++
+These commands create the following elements in the  `./optaplanner-quickstart` directory:
++
+//COMMENT please update
+** The Maven structure
```

Comment:
I don't think we need to add actual structure, do you?

Reply:
Ok, after reading the section it was not clear for me what is the final result we reaching.
Just because Maven structure is not the standard defenition
There is Standard Directory Layout - means "common directory layout allows users familiar with one Maven project to immediately feel at home in another Maven project."

Reply:
No, we do not necessarily need to write the directory tree